src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -1094,23 +1157,220 @@ class AflRunner(AflRunnerCommon, new_process.UnicodeProcessRunner):
   """Afl runner."""
 
   def __init__(self,
-               target_path,
-               config,
-               testcase_file_path,
-               input_directory,
-               timeout=None,
-               afl_tools_path=None,
-               strategy_dict=None):
+      target_path,
+      config,
+      testcase_file_path,
+      input_directory,
+      timeout=None,
+      afl_tools_path=None,
+      strategy_dict=None):
     super().__init__(target_path, config, testcase_file_path, input_directory,
                      timeout, afl_tools_path, strategy_dict)
 
     new_process.ProcessRunner.__init__(self, self.afl_fuzz_path)
 
 
+class AflAndroidRunner(AflRunnerCommon, new_process.UnicodeProcessRunner):
```

Comment:
A lot of this class is copied code. I wonder fi there is a better way to abstract things in the parent class so that the code can be reused.

Reply:
consolidated as much as possible.